src/components/__test__/app.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import App from '../../App';
+import { covidReducer } from '../../Redux/covid/covid';
+
+const reducer = combineReducers({
+  covidReducer,
+
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger, thunk),
+);
+
+describe('App', () => {
+  it('renders ! ', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <Router>
+          <App />
+        </Router>
+      </Provider>,
+    )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job writing some tests. However, it is required to write unit tests for several of your components but you have just two tests which are not enough as you have up to six components. Please consider adding more tests to cover most of your components.

- [ ] Currently, one of the two tests is failing. After debugging the cause of the failure, I found out that you spelled the redux module wrongly `Redux` instead of `redux` when you are importing the `covid` module. Please bear in mind that modules are case sensitive😉. Please make the correction so that the test will pass.

<img width="452" alt="Loom_EfC10HGjR4" src="https://user-images.githubusercontent.com/81468675/153694825-cea94862-cd72-4def-a46e-bfae3c33e006.png">